packages/graphback-codegen-client/src/templates/fragmentFields.ts

```diff
@@ -9,45 +10,43 @@ import { getNamedType, GraphQLObjectType, isCompositeType, isWrappingType } from
  * @param level nested level (supports only 0, 1 nested level)
  */
 export function buildReturnFields(t: GraphQLObjectType, level?: number) {
-    const fieldsMap = t.getFields();
-    if (level > 1) {
-      throw new Error("Function supports only 1 nested level")
-    }
-  
-    //tslint:disable-next-line: typedef
-    return Object.keys(fieldsMap).reduce((data, key) => {
-      const field = fieldsMap[key];
-      if (isWrappingType(field.type)) {
```

Comment:
@wtrocki I removed `isWrappingType` because I couldn't understand what it is for in this context.

Reply:
I have the same with our own relationship helpers ;)
Helpers are evil and some large amount of knowledge is lost because we have them. If we remove wrapping type we will apply NonNull and break stuff.

Reply:
Hmm I am not so sure about this. `getNamedType` unwraps a type so `modelType` is actually unwrapped, so there is no possibility of applying non-null.

Also fragments only used the `key` which never includes `!`

Reply:
This was to detect array.